examples/22_define-router/src/entries.tsx

```diff
@@ -193,4 +193,24 @@ export default defineRouter({
     }
     throw new Error('renderRoute: No such path:' + path);
   },
+  getApiConfig: async () => [
+    {
+      path: [
+        { type: 'literal', name: 'api' },
+        { type: 'literal', name: 'hi' },
+      ],
+      mode: 'static',
+      method: 'GET',
+      handler: async () => {
+        return new Response('Hello World');
+      },
```

Comment:
getApiConfig doesn't take these things. It's handleApi's job.

Reply:
Ah, right. Should the return type of body be anything that is json serializable then?

Right now it's `Record<string, unknown> | null` right?

Reply:
It's `ReadableStream | null`.